models/topics_models.js

```diff
@@ -5,4 +5,8 @@ function fetchTopics() {
     return require(`${__dirname}/../db/data/test-data/topics.js`)
 }
 
-module.exports = {fetchTopics}
\ No newline at end of file
+function fetchEndPoints() {
+    return require(`${__dirname}/../endpoints.json`)
```

Comment:
don't even really need a model for this endpoint as it's not interacting with database